csfieldguide/static/interactives/rgb-mixer/js/rgb-mixer.js

```diff
@@ -123,18 +130,34 @@ RGB_Mixer.setColor = function() {
   var b_val = RGB_Mixer.sliders[2].noUiSlider.get();
 	// stick them together.
   if (useHex) {
+    // Produce full hex code for displaying inside coloured box
+    hex_red = (r_val.length == 1) ? "0" + r_val : r_val;
+    hex_green = (g_val.length == 1) ? "0" + g_val : g_val;
+    hex_blue = (b_val.length == 1) ? "0" + b_val : b_val;
+    var colour_code = '#' + hex_red + hex_green + hex_blue;
+    // Convert values to decimal
     r_val = parseInt(r_val, 16);
     g_val = parseInt(g_val, 16);
     b_val = parseInt(b_val, 16);
+  } else {
+    // Produce full rgb code for displaying inside coloured box
+    var colour_code = 'rgb(' +
+      r_val + ',' +
+      g_val + ',' +
+      b_val + ')';
   }
+  var font_colour = getFontColour(r_val, g_val, b_val);
+  $('#interactive-rgb-mixer-colour-code')
+    .text(colour_code)
+    .css('color', font_colour);
```

Comment:
Note for the future: This should be much larger and in a monospaced font. I'll refrain from commenting further

![image](https://user-images.githubusercontent.com/25916475/88893065-e6aabd80-d299-11ea-9064-b95f4c9f94e0.png)